apps/scriptkit/src/certificate/certificate-handler.server.tsx

```diff
@@ -0,0 +1,208 @@
+// @ts-nocheck
```

Comment:
I had to do this in both places `@vercel/og` is used because the `tw` prop is a type error and I couldn't figure out the right switch to flip 😭